packages/ledger/src/lib/ledger.ts

```diff
@@ -1,15 +1,15 @@
 import { isMobile } from "is-mobile";
 import { TypedError } from "near-api-js/lib/utils/errors";
 import { signTransactions } from "@near-wallet-selector/wallet-utils";
-import type {
```

Comment:
We should maintain this import style here but have a new import statement for `getActiveAccount`

Reply:
Same for the other wallets